aspnetcore/fundamentals/index.md

```diff
@@ -4,17 +4,22 @@ author: rick-anderson
 description: Discover the foundational concepts for building ASP.NET Core apps.
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/20/2018
+ms.date: 10/25/2018
 uid: fundamentals/index
 ---
 # ASP.NET Core fundamentals
 
-An ASP.NET Core app is a console app that creates a web server in its `Main` method:
+An ASP.NET Core app is a console app that creates a web server in its `Program.Main` method. The `Main` method is the app's *managed entry point*:
 
 ::: moniker range=">= aspnetcore-2.0"
 
 [!code-csharp[](index/snapshots/2.x/Program.cs)]
 
+The .NET Core Host, which is usually the [dotnet utility](/dotnet/core/tools/dotnet):
+
+* Loads the [.NET Core Runtime](https://github.com/dotnet/coreclr).
+* Takes the first command line argument as the path to the managed binary that contains the managed entry point (`Main`) and begins code execution.
```

Comment:
Consider replacing "Takes" with something else like "Interprets" or "Uses"

Reply:
Yes, and I think drop the 2nd "managed."

If it's a managed binary, the entry point inside of it must be managed. This sentence is borderline *too long* anyway.